clang/test/SemaTemplate/address_space-dependent.cpp

```diff
@@ -43,7 +43,7 @@ void neg() {
 
 template <long int I>
 void tooBig() {
-  __attribute__((address_space(I))) int *bounds; // expected-error {{address space is larger than the maximum supported (8388586)}}
+  __attribute__((address_space(I))) int *bounds; // expected-error {{address space is larger than the maximum supported (8388585)}}
```

Comment:
Seems like because I use one more ID, the remaining space gets smaller.
If you look at this file history, each new address space added decreased this by 1.